src/Network/NWListener.cs

```diff
@@ -26,6 +26,8 @@ public enum NWListenerState {
 	[TV (12,0), Mac (10,14), iOS (12,0)]
 	[Watch (6,0)]
 	public class NWListener : NativeObject {
+		bool connectionHandlerWasSet = false;
+		object connectionHandlerLock = new object ();
```

Comment:
C# doesn't have a specific `Lock` type? We just create a generic `object` and use it with `lock` statements?

Reply:
Correct, it's possible to lock on any object.